src/bun.js/test/jest.zig

```diff
@@ -633,22 +633,23 @@ pub const TestScope = struct {
 
         if (JSC.getFunctionData(function)) |data| {
             var task = bun.cast(*TestRunnerTask, data);
+            const current_test = task.describe.tests.items[task.describe.current_test_id];
+            const is_failing = current_test.tag == .fail;
```

Comment:
nit, would name this like 'test_expects_failure' to avoid confusion with if the test itself failed or not (stored in test_succeeded)